app.js

```diff
@@ -25,18 +28,21 @@ const sessionSettings = {
 // register all partials from directory.
 hbs.registerPartials("./views/partials");
 
-app.set("views", "./views");
+app.set("views", path.join(__dirname, "views"));
```

Comment:
It is, but if you were executing it from somewhere else, it would not be in the current working directory.